tests/test-cases.js

```diff
@@ -1227,10 +1227,9 @@ var CUSTOM_TESTS = [
 				step: function(response, handle) {
 					updates.push(response.data[0]);
 					handle.abort();
-				},
-				complete: function() {
 					callback(updates);
```

Comment:
Matt told me at one point that he thinks the complete callback should still be sent after an abort. Would it be difficult to preserve that behavior?